helm: rook-ceph: helm lint is unable to parse rolebinding.yaml

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**

When I run `helm lint` on the `rook-ceph` chart I get the following error:

```txt
==> Linting rook-ceph
[ERROR] templates/rolebinding.yaml: unable to parse YAML: invalid Yaml document separator: kind: RoleBinding

Error: 1 chart(s) linted, 1 chart(s) failed
```

**Expected behavior:**

```txt
==> Linting rook-ceph

1 chart(s) linted, 0 chart(s) failed
```

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->

```sh
git checkout v1.8.3 && cd deploy/charts/ && helm lint rook-ceph
```

**Environment**:

```txt
$ lsb_release --all
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic

$ uname --kernel-name --kernel-release --machine
Linux 5.4.0-96-generic x86_64

$ helm version
version.BuildInfo{Version:"v3.7.2", GitCommit:"663a896f4a815053445eec4153677ddc24a0a361", GitTreeState:"clean", GoVersion:"go1.16.10"}

$ git show --oneline
dc06a70c8 (HEAD, tag: v1.8.3) Merge pull request #9635 from travisn/release-1.8.3
```

--- a/deploy/charts/library/templates/_cluster-psp.tpl
+++ b/deploy/charts/library/templates/_cluster-psp.tpl
@@ -61,4 +61,4 @@ subjects:
   - kind: ServiceAccount
     name: rook-ceph-cmd-reporter
     namespace: {{ .Release.Namespace }} # namespace:cluster
-{{- end -}}
+{{- end }}

--- a/deploy/charts/library/templates/_cluster-rolebinding.tpl
+++ b/deploy/charts/library/templates/_cluster-rolebinding.tpl
@@ -90,4 +90,4 @@ subjects:
   - kind: ServiceAccount
     name: rook-ceph-purge-osd
     namespace: {{ .Release.Namespace }} # namespace:cluster
-{{- end -}}
+{{- end }}

--- a/deploy/charts/library/templates/_recommended-labels.tpl
+++ b/deploy/charts/library/templates/_recommended-labels.tpl
@@ -6,4 +6,4 @@ app.kubernetes.io/part-of: rook-ceph-operator
 app.kubernetes.io/managed-by: helm
 app.kubernetes.io/created-by: helm
 helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
-{{- end -}}
+{{- end }}

--- a/deploy/charts/library/templates/_suffix-cluster-namespace.tpl
+++ b/deploy/charts/library/templates/_suffix-cluster-namespace.tpl
@@ -14,5 +14,5 @@ If the cluster namespace is different from the operator namespace, we want to na
 {{- $clusterNamespace := .Release.Namespace -}}
 {{- if ne $clusterNamespace $operatorNamespace -}}
 {{ printf "-%s" $clusterNamespace }}
-{{- end -}}
-{{- end -}}
+{{- end }}
+{{- end }}

--- a/deploy/charts/rook-ceph-cluster/templates/_helpers.tpl
+++ b/deploy/charts/rook-ceph-cluster/templates/_helpers.tpl
@@ -3,14 +3,14 @@ Define the clusterName as defaulting to the release namespace
 */}}
 {{- define "clusterName" -}}
 {{ .Values.clusterName | default .Release.Namespace }}
-{{- end -}}
+{{- end }}
 
 {{/*
 Return the target Kubernetes version.
 */}}
 {{- define "capabilities.kubeVersion" -}}
 {{- default .Capabilities.KubeVersion.Version .Values.kubeVersion -}}
-{{- end -}}
+{{- end }}
 
 {{/*
 Return the appropriate apiVersion for ingress.
@@ -22,5 +22,5 @@ Return the appropriate apiVersion for ingress.
 {{- print "networking.k8s.io/v1beta1" -}}
 {{- else -}}
 {{- print "networking.k8s.io/v1" -}}
-{{- end -}}
-{{- end -}}
+{{- end }}
+{{- end }}

--- a/deploy/charts/rook-ceph-cluster/templates/rbac.yaml
+++ b/deploy/charts/rook-ceph-cluster/templates/rbac.yaml
@@ -14,7 +14,7 @@ clusterrolebindings
 {{- if .Values.pspEnable }}
 ---
 {{ include "library.cluster.psp.rolebindings" . }}
-{{- end -}}
+{{- end }}
 
 {{/*
 roles
@@ -38,4 +38,4 @@ rolebindings
 {{ include "library.cluster.monitoring.rolebindings" . }}
 {{- end }}
 
-{{- end -}}
+{{- end }}

--- a/deploy/charts/rook-ceph/templates/cluster-rbac.yaml
+++ b/deploy/charts/rook-ceph/templates/cluster-rbac.yaml
@@ -18,7 +18,7 @@ clusterrolebindings
 {{- if .Values.pspEnable }}
 ---
 {{ include "library.cluster.psp.rolebindings" . }}
-{{- end -}}
+{{- end }}
 
 {{/*
 roles

--- a/deploy/charts/rook-ceph/templates/clusterrole.yaml
+++ b/deploy/charts/rook-ceph/templates/clusterrole.yaml
@@ -535,4 +535,4 @@ rules:
   - apiGroups: [""]
     resources: ["serviceaccounts"]
     verbs: ["get"]
-{{- end -}}
+{{- end }}

--- a/deploy/charts/rook-ceph/templates/clusterrolebinding.yaml
+++ b/deploy/charts/rook-ceph/templates/clusterrolebinding.yaml
@@ -99,4 +99,4 @@ roleRef:
   kind: ClusterRole
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
-{{- end -}}
+{{- end }}

--- a/deploy/charts/rook-ceph/templates/psp.yaml
+++ b/deploy/charts/rook-ceph/templates/psp.yaml
@@ -168,4 +168,4 @@ subjects:
     name: rook-csi-rbd-provisioner-sa
     namespace: {{ .Release.Namespace }} # namespace:operator
 {{- end }}
-{{- end -}}
+{{- end }}

--- a/deploy/charts/rook-ceph/templates/rolebinding.yaml
+++ b/deploy/charts/rook-ceph/templates/rolebinding.yaml
@@ -47,7 +47,7 @@ roleRef:
   name: rbd-csi-nodeplugin
   apiGroup: rbac.authorization.k8s.io
 ---
-{{- end -}}
+{{- end }}
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
@@ -61,4 +61,4 @@ roleRef:
   kind: Role
   name: rbd-external-provisioner-cfg
   apiGroup: rbac.authorization.k8s.io
-{{- end -}}
+{{- end }}

--- a/tests/scripts/helm.sh
+++ b/tests/scripts/helm.sh
@@ -2,7 +2,7 @@
 
 temp="/tmp/rook-tests-scripts-helm"
 
-helm_version="${HELM_VERSION:-"v3.6.2"}"
+helm_version="${HELM_VERSION:-"v3.8.0"}"
 arch="${ARCH:-}"
 
 detectArch() {

